<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldDialogUI</class>
 <widget class="QDialog" name="FieldDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form2</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel6">
       <property name="text">
        <string>Method Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Field Name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="textLabel3">
       <property name="text">
        <string>Field Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mMethod"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="mType">
       <item>
        <property name="text">
         <string>UInt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UInt8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>String</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Float</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Date</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DateTime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Time</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interval</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Primary Key</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Foreign Key</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ByteArray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Image</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="mDisplayName"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Display Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Plural Method:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="mPluralMethod"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="mLocal">
     <property name="text">
      <string>Local Variable</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mDefaultSelectCheck">
     <property name="text">
      <string>Default Select</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mListCheck">
     <property name="text">
      <string>Unique</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mDisplayNameCheck">
     <property name="text">
      <string>Table Display Name Field</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mIndexGroup">
     <property name="title">
      <string>Index</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="mUseCacheCheck">
        <property name="text">
         <string>Use Cache</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mDeleteModeCombo">
        <item>
         <property name="text">
          <string>Do Nothing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Update</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cascade</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>On Delete:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mIndexName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mFKGroup">
     <property name="title">
      <string>Foreign Key</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="mTableLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Table:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mTableCombo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="mReverseAccess">
        <property name="text">
         <string>Generate Reverse-Accessor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>345</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="mEditDocsButton">
     <property name="text">
      <string>Edit Documentation...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>41</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mName</tabstop>
  <tabstop>mMethod</tabstop>
  <tabstop>mPluralMethod</tabstop>
  <tabstop>mType</tabstop>
  <tabstop>mDisplayName</tabstop>
  <tabstop>mLocal</tabstop>
  <tabstop>mListCheck</tabstop>
  <tabstop>mDisplayNameCheck</tabstop>
  <tabstop>mIndexGroup</tabstop>
  <tabstop>mIndexName</tabstop>
  <tabstop>mDeleteModeCombo</tabstop>
  <tabstop>mUseCacheCheck</tabstop>
  <tabstop>mTableCombo</tabstop>
  <tabstop>mReverseAccess</tabstop>
  <tabstop>mEditDocsButton</tabstop>
  <tabstop>mOK</tabstop>
  <tabstop>mCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mCancel</sender>
   <signal>clicked()</signal>
   <receiver>FieldDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mOK</sender>
   <signal>clicked()</signal>
   <receiver>FieldDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
